aries/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+
+name = "aries"
+version = "0.1.0"
```

Comment:
I suppose it's good idea to version `libvcx` and `aries-vcx` separately, but maybe it'd be better to start with everything on the same version, and then decouple them as a next steps? It will def need  CI rethinking and changes